src/mcts.cpp

```diff
@@ -1312,7 +1312,7 @@ void SEARCHER::play_simulation(Node* n, float& score, int& visits) {
             /*Next ply depth*/
             pstack->depth = (pstack - 1)->depth - 3 - 
                             (pstack - 1)->depth / 4 -
-                            (MIN(3 , (n->score - (pstack - 1)->beta) / 128));
+                            (MIN_SCORPIO((float)3 , (n->score - (pstack - 1)->beta) / 128));
```

Comment:
This is supposed to be an integer calculation so maybe we can skip the cast to float ?

Reply:
Yes but both sides should have the same type ;) 
I think the right part is a float, if it makes more sense, you can cast the right side to an integer.